xpdacq/yamlclass.py

```diff
@@ -0,0 +1,64 @@
+##############################################################################
+#
+# xpdacq            by Billinge Group
+#                   Simon J. L. Billinge sb2896@columbia.edu
+#                   (c) 2016 trustees of Columbia University in the City
+#                   of
+#                        New York.
+#                   All rights reserved
+#
+# File coded by:    Timothy Liu
+#
+# See AUTHORS.txt for a list of people who contributed.
+# See LICENSE.txt for license information.
+#
+##############################################################################
+
+import os
+import yaml
+
+class YamlClass:
```

Comment:
No, it wouldn't be updated.

But why should we expect it to be updated? When you create a ``dict``, it is the value of input variable being passed, not the variable itself.

````
In [26]: word =1

In [27]: a = {'hello':word}

In [28]: a
Out[28]: {'hello': 1}
````

Is there any specific reason you want this logic to work? (``weakref``?)